src/FluxCalculator.jl

```diff
@@ -324,20 +277,22 @@ function compute_surface_fluxes!(
     Interfacer.get_field!(csf.scalar_temp4, sim, Val(:beta))
     beta = csf.scalar_temp4
 
-    scheme_properties =
-        (; z0b = z0b, z0m = z0m, Ch = FT(0), Cd = FT(0), beta = beta, gustiness = FT(1))
-
-    input_args = (;
-        thermo_state_sfc,
-        thermo_state_atmos,
-        uₕ_int,
-        csf.z_int,
-        csf.z_sfc,
-        scheme_properties,
-        boundary_space,
-        surface_params,
+    # Set some scalars that we hardcode for now
+    gustiness = FT(1)
+
+    # Construct the SurfaceFluxes.jl container of inputs
+    inputs = @. SF.ValuesOnly(
+        SF.StateValues(csf.z_int, uₕ_int, thermo_state_atmos), # state_in
+        SF.StateValues(                                  # state_sfc
+            csf.z_sfc,
+            StaticArrays.SVector(FT(0), FT(0)),
+            thermo_state_sfc,
+        ),
+        z0m,
+        z0b,
+        gustiness,
```

Comment:
`gustiness` and `beta` are `kwargs` within the existing constructors - so using the `;` separator should clarify this here.